Makefile

```diff
@@ -0,0 +1,38 @@
+default: test
+
+# Builds bosh-google-cpi for linux-amd64
+build:
+	env GOOS=linux GOARCH=amd64 go build -o out/cpi github.com/frodenas/bosh-google-cpi/main
```

Comment:
The old `build` script detected if the platform was `linux`, so it ran the `build-linux-amd64`, or, otherwise, it assumed it was not linux (like `Darwin`), and therefore, didn't add the `GOOS=linux`. Here seems to add  `GOOS=linux` on both cases, so it won't produce an executable compatible with `Darwin`.